packages/@react-spectrum/combobox/src/MobileComboBox.tsx

```diff
@@ -402,6 +419,22 @@ function ComboBoxTray(props: ComboBoxTrayProps) {
     lastInputValue.current = inputValue;
   }, [loadingState, inputValue, showLoading]);
 
+  let onKeyDown = (e) => {
+    // Close virtual keyboard if user hits Enter w/o any focused options
+    if (e.key === 'Enter' && state.selectionManager.focusedKey == null) {
+      inputRef.current.readOnly = true;
+      popoverRef.current.focus();
+
+      // Required to keep keyboard from reopening, 6ms is minimum time so that editing the field and hitting enter hides the virtual keyboard in iOS
+      setTimeout(() => {
+        inputRef.current.focus();
+        inputRef.current.readOnly = false;
+      }, 6);
```

Comment:
Is all of that really necessary? I tried it locally with only `popoverRef.current.focus()` and it seemed to work for me.

Reply:
Well, I wanted to keep the input field focused in case the user wanted to perform other operations (arrow keys to navigate through the options, typing, etc) but if I only have `popoverRef.current.focus()` they wouldn't be able to do that.

That being said, I don't know how many people use a keyboard with their phone haha, figured I'd play it safe.

Reply:
Ohh, didn't realize that. Yeah... I kinda doubt it. Also in that case, the software keyboard wouldn't even be visible in the first place. IMO we should keep it simple.

Reply:
Ohhhhh good point. Kk, will remove the extra stuff and only focus the popover